.pre-commit-config.yaml

```diff
@@ -1,100 +1,111 @@
+      - id: flake8
+        name: flake8
+        entry: flake8
+        language: system
+        types: [python]
+        log_file: '.artifacts/flake8.pycodestyle.log'
+      - id: lint-requirements
+        name: lint-requirements
+        entry: python -m tools.lint_requirements
+        language: python
+        files: requirements-.*\.txt$
+        # exclude the overrides file from this linter -- it doesn't apply at all
+        exclude: ^requirements-getsentry-overrides\.txt$
+        additional_dependencies: [packaging==21.3]
+      - id: check-mypy-bypass
+        name: do not bypass the type checker
+        entry: '(^# *mypy: *ignore-errors|^# *type: *ignore|\bno_type_check\b)'
+        language: pygrep
+        types: [python]
+        exclude: ^src/sentry/metrics/minimetrics.py$
+      - id: prevent-push
+        name: prevent pushing master
+        stages: [pre-push]
+        entry: bash -c 'test "$PRE_COMMIT_REMOTE_BRANCH" != "refs/heads/master"'
+        always_run: true
+        pass_filenames: false
+        language: system
 
--   repo: https://github.com/pre-commit/pygrep-hooks
+  - repo: https://github.com/pre-commit/pygrep-hooks
     rev: v1.10.0
     hooks:
-    -   id: python-use-type-annotations
+      - id: python-use-type-annotations
 
--   repo: https://github.com/python-jsonschema/check-jsonschema
+  - repo: https://github.com/python-jsonschema/check-jsonschema
     rev: 0.24.1
     hooks:
-    - id: check-github-actions
-    - id: check-github-workflows
-      args: [--verbose]
+      - id: check-github-actions
+      - id: check-github-workflows
+        args: [--verbose]
 
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.3.0
     hooks:
-    - id: check-case-conflict
-    - id: check-executables-have-shebangs
-    - id: check-merge-conflict
-    - id: check-symlinks
-    - id: check-yaml
-    - id: end-of-file-fixer
-      exclude_types: [svg]
-      exclude: ^fixtures/
-    - id: trailing-whitespace
-      exclude_types: [svg]
-      exclude: ^(fixtures/|scripts/patches/)
-    - id: debug-statements
-    - id: name-tests-test
-      args: [--pytest-test-first]
+      - id: check-case-conflict
+      - id: check-executables-have-shebangs
+      - id: check-merge-conflict
+      - id: check-symlinks
+      - id: check-yaml
+      - id: end-of-file-fixer
+        exclude_types: [svg]
+        exclude: ^fixtures/
+      - id: trailing-whitespace
+        exclude_types: [svg]
+        exclude: ^(fixtures/|scripts/patches/)
+      - id: debug-statements
+      - id: name-tests-test
+        args: [--pytest-test-first]
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: 'v3.0.3' # Use the sha or tag you want to point at
+    hooks:
+      - id: prettier
+        name: prettier (yaml, json, markdown)
+        types_or:
+          - json
+          - yaml
+          - markdown
+        exclude_types: [binary]
```

Comment:
this doesn't do anything because `binary` is not included in `types_or`

Reply:
Fixed, now.